runtime/lua/vim/shared.lua

```diff
@@ -347,6 +347,30 @@ function vim.tbl_add_reverse_lookup(o)
   return o
 end
 
+--- Index into a table using a list of keys. Return `nil` if the key does not exist.
+--- Examples:
+--- <pre>
+---  vim.tbl_get({ key = { nested_key = true }}, {'key', 'nested_key'}) == true
+---  vim.tbl_get({ key = {}}, {'key', 'nested_key'}) == nil
+--- </pre>
+---
+---@param o Table to index
+---@param keys List of indices via which to index the table
+---
+---@returns nested value indexed by key if it exists, else nil
+function vim.tbl_get(o, keys)
+  if #keys == 0 then
```

Comment:
You might want to add a nil check here too:

```suggestion
  if not keys or #keys == 0 then
```